assets/app/scripts/controllers/builds.js

```diff
@@ -21,6 +22,9 @@ angular.module('openshiftConsole')
     var watches = [];
 
     watches.push(DataService.watch("builds", $scope, function(builds, action, build) {
+      var buildConfigName;
+      var buildName;
```

Comment:
I'm ok with declaring the vars at the top scope, but can we keep them close to the block where they are defined, rather than hoisting to the top of the function? We should only be referencing them after that block anyway